Dockerfile

```diff
@@ -32,6 +32,17 @@ RUN apt-get install -y apt-utils \
 
 RUN ln -s /bin/mkdir /usr/bin/mkdir
 
+# Run tests on a module created with PDK 1.18.1 using the current PDK to pull in
+# any other dependencies and then delete the 1.18.1 test module.
+#
+# Simply running "bundle install" against the module is not enough, 
+# as PDK has further dependencies to pull in.
+COPY pdk_1_18_1_dependencies /test_module
+WORKDIR /test_module
```

Comment:
I would just do a `cd`/`pushd` in the `RUN` command below instead of permanently changing the image's `WORKDIR`.

This `WORKDIR` statement will cause all subsequent commands in the Dockerfile to run in this directory, which will have been `rm -rf`'d. You would want to un-set the WORKDIR so it goes back to where it was (where was that? ;)) .... or just avoid it and do the `cd`/`pushd`+`popd` all in the one-liner.